src/encoded/static/components/filegallery.js

```diff
@@ -275,6 +282,8 @@ FileTable.propTypes = {
     setInfoNodeVisible: PropTypes.func,
     /** User session */
     session: PropTypes.object,
+    /** User session properties */
+    session_properties: PropTypes.object,
```

Comment:
ESLint error. You’re getting session_properties from contextTypes, so you don’t use this one.

Reply:
Fixed. I did not get the exception initially.